src/dotnet-install.sh

```diff
@@ -423,11 +423,17 @@ get_normalized_architecture_for_specific_sdk_version() {
 # args:
 # version or channel - $1
 is_arm64_supported() {
-    #any channel or version that starts with the specified versions
-    case "$1" in
-        ( "1"* | "2"* | "3"*  | "4"* | "5"*) 
-            echo false
-            return 0
+    # Extract the major version by splitting on the dot
+    major_version="${1%%.*}"
```

Comment:
I guess this will have to be updated again in 10 years, for .NET 20 :smile:

Reply:
Haha, I see what you're saying! But no worries, since we're already checking for versions less than six, the script will handle .NET 20 and beyond just fine.

I think there might be a bit of confusion about `${1}`, it’s actually referring to the first positional argument passed into the function, not the number `1` as the major version. To make things clearer, maybe we could assign `$1` to a more descriptive variable like `version_string` and then extract the major version from there. Something like:

```bash
version_string="$1"
major_version="${version_string%%.*}"
```

Just a thought for better readability! 😊

Reply:
> > I guess this will have to be updated again in 10 years, for .NET 20 😄
> 
> the 1 in ${1%%.*} is arg number not the version standard shell syntax
> 

definitely not my sharpest moment... 🤦‍♂️